pushToTamrNexus.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -e
+
+function fullfile {
+    echo $(cd $(dirname $1) && pwd)/$(basename $1)
+}
+
+function pushjar {
+    if [ $# -lt 4 ]; then
+        echo "usage: $0 <file> <groupId> <artifactId> <version>"
+        return 1
+    fi
+    mvn deploy:deploy-file -Dfile=$(fullfile $1) \
```

Comment:
Might want to check for Maven version being some minimum. Older versions won't do the right thing.